### PR TITLE
Update on fund details page

### PR DESCRIPTION
**Pull request by line47**
This update includes work on the tabs section at the bottom of the page.

**Screenshots**
screen shot 2018-08-07 at 9 41 02 am
screen shot 2018-08-07 at 9 41 09 am
screen shot 2018-08-07 at 9 41 15 am

### CODE REVIEW

File: _layouts/fund-details.html [hunk 1]
```diff
@@ -69,7 +69,81 @@ <h2>Performance</h2>
 </table>
 
 </div>
+
+<ul class="usa-accordion usa-tabs">
+    <li>
```
Comment (line47) [inline _layouts/fund-details.html]: For the tabs the first and last tab content comes from front matter and the 2 middle tabs with charts and graphs will come from the content of the page layout

File: _layouts/fund-details.html [hunk 2]
```diff
@@ -69,7 +69,81 @@ <h2>Performance</h2>
...
+                  {{ item.abbr }}
+                  {{ item.name }}
+              </li>
+              {% endfor %}
+            </ol>
+            </div>
+          </div>
+        </div>
+    </li>
   {{ content }}
```
Comment (line47) [inline _layouts/fund-details.html]: Middle tabs comes from the page content

File: pages/fund-basics/s-fund.md [hunk 3]
```diff
@@ -22,41 +22,68 @@ avg_annual_returns:
     three_yr: +14.89%
     five_yr: +15.89%
     ten_yr: +16.89%
+summary_details:
```
Comment (line47) [inline pages/fund-basics/s-fund.md]: See below the additional front matter added

**Comment by chachi69** [comment]
@line47 Okay to merge.

**Pull request merged**